src/app/board/question/bounty/page.tsx

```
@@ -0,0 +1,148 @@
+        pageNumber: page,
+        pageSize: 10,
+        sortType: tabType === "최근순" ? "LATEST" : "REWARD_YEOPJEON_DESCENDING", // 엽전 내림차순 정렬
+        questionPresetTags: options.qnaPresetTags,
+        chaetaekStatus: options.chaetaekStatus,
+        // minYeopjeon: 1, // 최소 엽전 현상금 1 이상 - 필요시 추가
+      });
+
+      if (response && response.questionPostsPage) {
+        // 현상금이 있는 질문만 필터링
+        const bountyQuestions = (response.questionPostsPage.content || []).filter(
+          question => question.rewardYeopjeon && question.rewardYeopjeon > 0,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)